qemu_v8.mk

```diff
@@ -303,6 +331,44 @@ urootfs: buildroot $(BINARIES_PATH)
 				-n "Root file system" \
 				-d $(ROOTFS_GZ) $(ROOTFS_UGZ)
 
+################################################################################
+# XEN
+################################################################################
+.PHONY: xen
+xen-defconfig:
+	$(MAKE) -C $(XEN_PATH)/xen XEN_TARGET_ARCH=arm64 defconfig
+
+xen-common: xen-defconfig
+	cd $(XEN_PATH)/xen && \
+	tools/kconfig/merge_config.sh -m .config $(ROOT)/build/kconfigs/xen.conf
+
+xen: xen-common $(BINARIES_PATH)
+	$(MAKE) -C $(XEN_PATH) dist-xen \
+	XEN_TARGET_ARCH=arm64 \
+	CONFIG_XEN_INSTALL_SUFFIX=.gz	\
+	CROSS_COMPILE="$(CCACHE)$(AARCH64_CROSS_COMPILE)"
+	ln -sf $(XEN_PATH)/xen $(BINARIES_PATH)
+
+XEN_TMP ?= $(BINARIES_PATH)/xen_files
+
+# When creating the image containing Linux kernel, we need to temporarily store
+# the files somewhere.
+$(XEN_TMP):
+	mkdir -p $@
```

Comment:
ok

Reply:
Another option would be to use an order-only prerequisite:
```
$(XEN_TMP):
	mkdir $@

xen-create-image: xen linux buildroot | $(XEN_TMP)
	...
```
See https://www.gnu.org/software/make/manual/html_node/Prerequisite-Types.html.
That's a nice trick I think, worth remembering ;) that being said no reason to introduce this pattern here since we're using unconditional `mkdir -p`'s everywhere in the OP-TEE projects.

Reply:
Thanks @jforissier. This is indeed nice trick to remember